scripts/xdist/get_worker_test_list.py

```diff
@@ -32,12 +32,12 @@ def main(log_file, test_suite):
     worker_test_dict = {}
     with io.open(log_file, 'r') as console_file:
         for line in console_file:
-            regex_search = re.search(r'\[gw(\d+)] (PASSED|FAILED|SKIPPED|ERROR)'.format(test_suite), line)
+            regex_search = re.search(r'\[gw(\d+)] (PASSED|FAILED|SKIPPED|ERROR) (\S+)\b'.format(test_suite), line)
```

Comment:
Looks good! Could even leave off the `\b`, I bet—either end of line or the first whitespace would count as a word boundary anyhow.

Reply:
True.  I had a different issue at first and didn't realize I no longer needed this.